### PR TITLE
Added command `create-config` (#169)

### LINKED ISSUE #169: config.yml generator, command
Create a command to ask the user the configuration that he wants and generate a config.yml

Basic Flask config:
```
  config:
    DEBUG: true
    TESTING: false
    APP_NAME: "Python Microservice"
    APPLICATION_ROOT: ""
```

Input: do you want to use request? (Y/n) Y
```
pyms:
  services:
    requests:
      data: ""
  config:
    DEBUG: true
    TESTING: false
    APP_NAME: "Python Microservice"
    APPLICATION_ROOT: ""
```

Input: do you want to use swagger? (Y/n) Y
```
pyms:
  services:
    requests:
      data: ""
    swagger:
      path: ""
      file: "swagger.yaml"
  config:
    DEBUG: true
    TESTING: false
    APP_NAME: "Python Microservice"
    APPLICATION_ROOT: ""
```

### PR DESCRIPTION
**Pull request by path2himanshu**
- Added command functionality
- Added tests for command
- Added methods to dump, remove config file
NOTE: The command by default, doesn't create/overwrite if a config file
already exists.

Fixes #169 

Changes proposed in this PR:
- Added yaml config generator command

### CODE REVIEW

**Comment by alexppg** [comment]
@path2himanshu the coverage has decreased, can you increase it or level it with how it was?

**Comment by avara1986** [comment]
> @path2himanshu the coverage has decreased, can you increase it or level it with how it was?

And fix the linters :stuck_out_tongue:

**Comment by path2himanshu** [comment]
@alexppg the test coverage is back to normal now.
@avara1986 The linter breaking the build is will require to change the Command.__init__function to a major extent as the error is that function is too complex or we can make flake8 ignore it.

**Comment by path2himanshu** [comment]
@avara1986 Please review

File: pyms/config/conf.py [hunk 1]
```diff
@@ -144,3 +146,61 @@ def __verify_deprecated_env_variables(config):
         except AttrDoesNotExistException:
             pass
         logger.warning(msg)
+
+
+def create_conf_file(use_requests: bool = False, use_swagger: bool = False) -> Union[Exception, str]:  # pragma: no cover
```
Comment (avara1986) [inline pyms/config/conf.py:151]: remove `# pragma: no cover` and add tests

File: pyms/config/conf.py [hunk 2]
```diff
@@ -144,3 +146,61 @@ def __verify_deprecated_env_variables(config):
...
+    }
+    try:
+        with open(CONFIG_FILE, 'w', encoding='utf-8') as config_file:
+            config_file.write(yaml.dump(config, default_flow_style=False, default_style=None, sort_keys=False))
+    except Exception as ex:
+        raise ex
+    return CONFIG_FILE
+
+
+def remove_conf_file():  # pragma: no cover
```
Comment (avara1986) [inline pyms/config/conf.py:197]: is remove_conf_file only used in tests? could be and extra option to force create the file or move this fuction to tests

**Review by avara1986 — CHANGES_REQUESTED** [review]

**Comment by avara1986** [comment]
thanks for the PR @path2himanshu! I test it manually and works very well :smile: but the PR need more tests, especially in the `pyms/config/conf.py` file :+1:

File: pyms/config/conf.py (hunk 1 above)
Comment (path2himanshu) [reply pyms/config/conf.py]: Added test for create_conf_file

File: pyms/config/conf.py (hunk 2 above)
Comment (path2himanshu) [reply pyms/config/conf.py]: Moved remove_conf file to tests.common

**Review by avara1986 — APPROVED** [review]

**Pull request merged**